CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+CHANGES IN VERSION 1.8.0 (from 1.7.2)
+===================================
+
+Appium 1.8.0 introduces full support for the [W3C WebDriver specification](https://www.w3.org/TR/webdriver/)
+([digested version of the spec](https://github.com/jlipps/simple-wd-spec)).
+
+#### General
+* Minimum [NodeJS]() version moved to 6
+* Add methods for
+  * getting and setting the clipboard contents
+  * get full screen screenshots
+
+
+#### iOS
+* Support for iOS 11.3/Xcode 9.3
+* Fix handling of process arguments
+* Add capabilities
+  * `shutdownOtherSimulators` - shutdown other running simulators at session start up
+* Add ability to record screen
+* Fix handling of custom SSL certificate on simulators
+* Better handling of `xcodebuild` processes
+* Maintain keychains while upgrading apps
+* Better handle simulators in unexpected states
```

Comment:
Can we address measuring performance feature as well?

- https://github.com/appium/appium-xcuitest-driver/pull/637
- https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/ios/ios-xctest-performance.md